pkg/cmd/flags.go

```diff
@@ -329,6 +329,53 @@ func AddProviderFlag(cmd *cobra.Command, command *AuthenticatedCLICommand) {
 	})
 }
 
+func AddResourceOwnerFlag(cmd *cobra.Command, command *AuthenticatedCLICommand) {
+	items := []string{"user", "group-mapping", "service-account", "identity-pool"}
+	cmd.Flags().String("resource-owner", "",
+		fmt.Sprintf("The resource ID of the principal who will be assigned resource owner on the "+
+			"created resource. Principal can be a %s.", utils.ArrayToCommaDelimitedString(items, "or")))
```

Comment:
Can you split this out of the parent function call and put it in a variable called `description` for example? It's a bit confusing to read.